src/App/Fossa/Main.hs

```diff
@@ -107,6 +110,25 @@ experimentalLicenseScanCommand = command "experimental-license-scan" (info runIn
     runInit :: Parser (IO ())
     runInit = pure $ putStrLn "The 'experimental-license-scan' has been deprecated and renamed to 'license-scan'. Please use the 'license-scan' command instead."
 
+feedbackCommand :: Mod CommandFields (IO ())
+feedbackCommand = command "experimental-feedback" (info feedbackPrompt $ progDescDoc $ formatStringToDoc "Provide feedback on your FOSSA CLI experience, submit feature requests, and report bugs/issues")
+  where
+    feedbackPrompt :: Parser (IO ())
+    feedbackPrompt =
```

Comment:
Related to https://github.com/fossas/fossa-cli/pull/1368/files#r1478940928 - consider testing whether in a TTY and explicitly exiting with an error code if not. 

This way the only way a user can run it in a script is if they intentionally pretend to be in a TTY, ensuring that we don't have to consider scripting backwards compatibility ever for this subcommand.